src/components/VideoChatButtonAndMenu.js

```diff
@@ -63,11 +65,14 @@ class VideoChatButtonAndMenu extends Component {
     }
 
     componentDidMount() {
-        Dimensions.addEventListener('change', this.measureVideoChatIconPosition);
+        this.dimensionsEventListener = Dimensions.addEventListener('change', this.measureVideoChatIconPosition);
     }
 
     componentWillUnmount() {
-        Dimensions.removeEventListener('change', this.measureVideoChatIconPosition);
+        if (!this.dimensionsEventListener) {
+            return;
+        }
+        this.dimensionsEventListener.remove();
```

Comment:
NAB: The early return seems unnecessary in this case. We only run one line after the early return, and the condition is not business logic, so we may as well invert the condition and remove the listener if the condition is true.

Reply:
We set up a custom ESLint rule to enforce early returns so this would make the linter complain.

Reply:
Ah fair enough!